pkg/commands/run.go

```diff
@@ -466,7 +466,11 @@ func (c *runCommand) createPrinter(format string, w io.Writer) (printers.Printer
 }
 
 func (c *runCommand) printStats(issues []result.Issue) {
-	if !c.cfg.Run.ShowStats {
+	if c.cfg.Run.ShowStats {
+		c.log.Warnf("run.show-stats is deprecated, please use output.show-stats")
```

Comment:
Maybe it makes sense to add the prefix "The config option " for better clarity and embrace with backticks for consistency with this [warning](https://github.com/golangci/golangci-lint/blob/b14d05cdb451c9b51c6705401ffad5c7a6d11a46/pkg/golinters/unparam.go#L52C1-L53C1):

So, we get the following warning message:
<img width="764" alt="image" src="https://github.com/golangci/golangci-lint/assets/3228886/f1ec0fe1-d97a-4736-8dbb-78044632a00a">